portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,10 @@
  */
 function addRandomGreeting() {
```

Comment:
Maybe update "greeting" to "fact" here and below. Prefer meaningful variable and function names that tell what data they hold and what they do.